lib/Mojolicious/Guides/FAQ.pod

```diff
@@ -207,6 +207,30 @@ weakened.
     $c->reply->exception($err);
   });
 
+=head2 What does "Illegal character in prototype" mean?
+
+Mojolicious assumes L<subroutine signatures|Mojolicious::Guides/"Signatures"> are enabled in documentation examples. If
+the signatures feature has not been enabled in that scope, they are interpreted as L<prototypes|perlsub/"Prototypes">,
+an unrelated parser feature. The signatures feature is not required to use any Mojo tool; if you don't want to or
```

Comment:
`Mojo tool` sounds weird.

Reply:
Should probably say `Mojo feature`.

Reply:
I'd just rephrase. `Mojolicious does not require signatures` possibly with a leading clause like `However, `